lib/SystemStatistics.php

```diff
@@ -56,6 +56,11 @@ public function getSystemStatistics(): array {
 		$memoryUsage = $this->os->getMemory();
 		return [
 			'version' => $this->config->getSystemValue('version'),
+			'update' => [
+				'available' => count(json_decode($this->config->getAppValue('core', 'lastupdateResult'))) > 0 ? 'yes' : 'no',
```

Comment:
`json_decode`needs to have the the associated parameter set to true, otherwise it will return an object. Doubtful whether count() works there.  `\stdClass` is not documented as `Countable`. Valid also in the next line.

Reply:
Update invocation on line 69 with flag to return the array. The second invocation already has that flag